src/test/kotlin/com/charleskorn/kaml/YamlTaggedTest.kt

```diff
@@ -0,0 +1,105 @@
+/*
+
+   Copyright 2018-2019 Charles Korn.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+package com.charleskorn.kaml
+
+import ch.tutteli.atrium.api.cc.en_GB.notToThrow
+import ch.tutteli.atrium.api.cc.en_GB.toBe
+import ch.tutteli.atrium.verbs.assert
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+object YamlTaggedTest : Spek({
+    describe("a YAML tagged node") {
+        describe("creating an instance") {
```

Comment:
This kind of test is only necessary when we do some kind of validation in the constructor (eg. line we do in `YamlMap`)